meshdb/meshdb.go

```diff
@@ -201,18 +265,142 @@ func (m *MeshDB) FindLatestMiniHeader() (*MiniHeader, error) {
 	return miniHeaders[0], nil
 }
 
+// InsertOrder atomically inserts the given order and updates any relevant ETH
+// backings. It also removes the order with the lowest ETH backing if needed in
+// order to make room. If order storage is full and the given order does not
+// have a high enough ETH backing, it will not be stored (this is not considered
+// an error). InsertOrder will return an error if any corresponding ETH backings
+// cannot be found.
+// TODO: return hash of order that was deleted.
+func (m *MeshDB) InsertOrder(order *Order) error {
```

Comment:
@fabioberger we need to eventually return the order hash for any order that was deleted so that we can send an update to RPC clients. Are you okay with punting on this for now? It likely won't come up in the near future due to the fact that the total number of current open order book orders is much lower than the default limit.

Reply:
Yep, we can punt but can you please open an issue so we don't forget?